config/preset/theme.toml

```diff
@@ -1,6 +1,7 @@
 [tab]
-active   = { fg = "#1E2031", bg = "#80AEFA" }
-inactive = { fg = "#C8D3F8", bg = "#484D66" }
+active    = { fg = "#1E2031", bg = "#80AEFA" }
+inactive  = { fg = "#C8D3F8", bg = "#484D66" }
+max_width = 0
```

Comment:
I think `max_width` should represent the overall width of the tab (with default 1), not just the dir name, since it seems not related to the dir name by looking

Reply:
I agree. Do you have any suggestions about the naming?

Reply:
I prefer to keep this name, but change the behavior to fit it, so that it can cover what is added in the future not only the dir name currently.